ffs/api/istore.go

```diff
@@ -13,7 +13,7 @@ import (
 
 var (
 	dsDefaultConfig = datastore.NewKey("defaultconfig")
-	dsCidConfig     = datastore.NewKey("cidconfig")
+	dsStorageConfig = datastore.NewKey("cidconfig")
```

Comment:
If we decide not to migrate data, I'll update the key name, but leaving it for now.